src/Directory.Build.props

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildProjectDirectory), dir.props))\dir.props" />
+  <PropertyGroup>
+    <EnableDefaultItems>false</EnableDefaultItems>
+    <NoWarn>CS3021;$(NoWarn)</NoWarn>
+    <RestoreOutputPath>$(MSBuildThisFileDirectory)\System.Runtime\ref\</RestoreOutputPath>
+    <IsSDKProject>true</IsSDKProject>
+  </PropertyGroup>
+  <PropertyGroup>
+    <CSharpCoreTargetsPath>$(MSBuildExtensionsPath32)\Roslyn\Microsoft.CSharp.Core.targets</CSharpCoreTargetsPath>
```

Comment:
Why do we need to explicitly set these here?

Reply:
This is also a remnant I took from Jose's original changeset - I haven't tried running a build without it yet.